README.md

```diff
@@ -100,5 +109,18 @@ Synthetic training data is generated using the code in src/s2_fos/training/open_
 Can be found in [allenai/fos_model_training_data_open_ai_annotations](https://huggingface.co/datasets/allenai/fos_model_training_data_open_ai_annotations) under fos_open_ai_labels.parquet
 
 ## Fine-tuned model weights
-Fine-tuned model weights are available at [allenai/scibert_scivocab_uncased_fielf_of_study](https://huggingface.co/allenai/scibert_scivocab_uncased_fielf_of_study)
+Fine-tuned model weights are available at [allenai/scibert_scivocab_uncased_field_of_study](https://huggingface.co/allenai/scibert_scivocab_uncased_field_of_study)
 
+## How to fine-tune the model
+
+To fine tune the model 
+- Download the training data from [allenai/fos_model_training_data_open_ai_annotations](https://huggingface.co/datasets/allenai/fos_model_training_data_open_ai_annotations) under fos_open_ai_labels.parquet
+- Split the data into train, test and validation sets save them into separate parquet files
```

Comment:
We should do this for people! Please replace this file with a standard 3-file split and upload those instead.

Reply:
So what I did is am using now datasets api from hugging face to download data and split it into train/test/validation. If train/test/validation path is not provided.

What happens behind the scene, given hf_token of the user who agreed to the Ai2 impact license data set is downloaded and is split into train/test/validation if path to training/test/validation is None.

This should make it work out of the box for users and avoids manual steps of downloading the data.

Reply:
Got it. In that case, get rid of this instruction to split. Make it straightforward and unambiguous as possible.